.vscode/launch.json

```diff
@@ -16,6 +16,7 @@
       "request": "launch",
       "runtimeExecutable": "${execPath}",
       "args": [
+        // "--user-data-dir=${workspaceFolder}/target/code",
```

Comment:
What is this option for? Does this overwrite the `globalStoragePath`?

Reply:
Yup, this overrides storage path, extensions and settings, and it makes it easier to look inside the store. It’s deliberately commented out, so that it can be easily restored when needed